lib/commands/build.js

```diff
@@ -10,9 +9,9 @@ module.exports = Command.extend({
   aliases: ['b'],
 
   availableOptions: [
-    { name: 'environment', type: String,  default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
-    { name: 'output-path', type: path,    default: 'dist/',       aliases: ['o'] },
-    { name: 'watch',       type: Boolean, default: false,         aliases: ['w'] },
+    { name: 'environment', type: String,             default: 'development', aliases: ['e', { 'dev': 'development' }, { 'prod': 'production' }] },
+    { name: 'output-path', type: function Path() {}, default: 'dist/',       aliases: ['o'] },
```

Comment:
ok i will do it like this.
but just to show you what i prepared with the other approach:

utilities/option-type.js:
```javascript
'use strict';

module.exports = function(name) {
  return new Function('return function ' + name + '() {};')();
};
```

this way we could not only do: `type: optionType('Path')` but also `type: optionType('Url')` or  `type: optionType('FooBar')` if we want to in the future.

I don't know whether this type of meta programming is a bad practice though.

any thoughts? otherwise i will implement it the way you proposed as `utilities/path-option-type.js`